packages/landing/src/components/map.tsx

```diff
@@ -112,7 +112,9 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
     const center = this.map.getCenter();
     if (center == null) throw new Error('Invalid Map location');
     const zoom = Math.floor((this.map.getZoom() ?? 0) * 10e3) / 10e3;
-    return Config.map.transformLocation(center.lat, center.lng, zoom);
+    const location = { lat: center.lat, lon: center.lng, zoom };
+    Config.map.setLocation(location);
```

Comment:
I dont think a `getter` should also `set`, I think we only use this function in one place did you want to just inline it into the `setLocationUrl` function